components/generic/SimilarItems.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <section class="row py-4 px-2 bg-white">
```

Comment:
The similar items are going to be in a card. So could you remove the bg-white class and add the card class instead. 
`<section class="row py-4 px-2 card">`

Reply:
done